.github/actions/cargo-build-macos-binary/action.yml

```diff
@@ -61,11 +61,11 @@ runs:
         path: ./${{ env.ASSET_FULL_NAME }}.tar.gz
         retention-days: 3
     - name: Deploy archive to GitHub release
-      if: "${{ inputs.version != 'nightly' }}"
       uses: quickwit-inc/upload-to-github-release@v1
       env:
         GITHUB_TOKEN: ${{ inputs.token }}
       with:
         file: ${{ env.ASSET_FULL_NAME }}.tar.gz
         overwrite: true
+        draft: false
```

Comment:
Ok so if you remove the draft, then it's public.

This is fine for a nightly build but for a real release, we should keep `draft: true` rigth?

Reply:
I have fixed it. it's rather verbose which I was trying to avoid. Ideally, I need an action that just leaves the state of the release as is. Given the action is our own, can we update it?